text/0750-feature-flag-advisor.md

```diff
@@ -0,0 +1,183 @@
+# RFC: Feature Flag Advisor
+
+* Original Author(s): [Vivian Chen](https://quip-amazon.com/GGQ9EAAQ3Jw)
+* Tracking Issue: N/A
+* API Bar Raiser: [Rico Huijbers](https://quip-amazon.com/YPP9EA8Gt6Q)
+
+The feature flag CLI tool will inform users of their current feature flag configuration and allow them to switch any/all of them to a different value.
+
+## Working Backwards
+
+### **Help:** 
+
+```
+> cdk flags --help
+cdk flags 
+Finds and displays a report of the feature flag configuration and compares users' current values with our recommended values.
+ 
+Options:
+    --switch-all          **safely** switch all feature flags to their recommended states 
```

Comment:
"switch" also sounds slightly vague. 

`--make-all-recommended` ? Slightly long...

Reply:
Is there also an unsafe variant for this?

By the way, I think the use case for this operation is fringe. I think `--switch-unconfigured` and its converse, `--record-unconfigured` are the most useful ones.

I wonder if `--safe` should just be an additional flag. Or its inverse: `--allow-changes`.